src/screens/FeedScreen/index.js

```diff
@@ -257,7 +249,10 @@ const FeedScreen = (props) => {
         refreshing={loading}
         renderItem={renderItem}
         onEndReachedThreshold={0.9}
-        onMomentumScrollEnd={onWillSendViewPostTime}
+        onMomentumScrollEnd={(momentumEvent) => {
+          onWillSendViewPostTime(momentumEvent, feeds);
+          fetchNextFeeds(momentumEvent);
+        }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onMomentumScrollEnd` event handler now calls both `onWillSendViewPostTime` and `fetchNextFeeds`. This seems to be part of the refactoring effort to improve the handling of post view times and feed fetching. Ensure that this change doesn't introduce any unexpected behavior.
```diff
-        onMomentumScrollEnd={onWillSendViewPostTime}
+        onMomentumScrollEnd={(momentumEvent) => {
+          onWillSendViewPostTime(momentumEvent, feeds);
+          fetchNextFeeds(momentumEvent);
+        }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->